pages/index.js

```diff
@@ -71,42 +71,50 @@ export default function Home() {
 					vs_currencies: ['usd'],
 			});
 
-			setPrice(data.vechain.usd)
+			setVetPrice(data.vechain.usd)
 		}
 
     getPrice()
 	}, [])
 
 	useEffect(() => {
-    const getData = () => {
+      if (!vetPrice || tokens.length > 0) return;
       const promises = allTokens.map(async (item) => {
         const { data } = await axios.get(`/api/${item.address}`)
         item = { ...data, ...item }
 
+        // Multiplied by two because it's a 50-50 pool
+        item.tvlInUsd = item.reserves * item.price.base2quote * vetPrice * 2;
+
+        // Extra attribute for sorting API
+        // As sorting API cannot handle nested objects
+        item.priceInVet = item.price.base2quote;
+
+        item.annualizedFeeApr = item.volumeInVet * vetPrice * 0.01 // Fee generated in a day, currently hardcoded to 1%
```

Comment:
to be more accurate we should hardcode to 0.75% due to the current 25% platform fee

Reply:
```suggestion
        item.annualizedFeeApr = item.volumeInVet * 0.01 // Fee generated in a day, currently hardcoded to 1%
```

Reply:
okay got it. will change to 0.75. Thanks